test/smoketest/README.md

```diff
@@ -0,0 +1,147 @@
+# Smoketest of RLA server
+
+This smoketest has been tested against master commit 54a95d4.
+
+For now, you'll need to install
+[zerotest](https://github.com/jjyr/zerotest)
+to run these tests, and/or generate new ones in the same way.
+
+This is mainly tested with python3, but has been seen to work
+on python2 also.
+
+`pip install zerotest`
```

Comment:
... or 'pip3 install zerotest` (or similar) if you have multiple Python installations